utils/automation/FoScanner/FormatResults.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2024 Rajul Jha <rajuljha49@gmail.com>
+
+# SPDX-License-Identifier: GPL-2.0-only
+
+import re
+import os
+import binascii
+
+from .CliOptions import CliOptions
+
+class FormatResult:
+    """
+    For formatting the results from scanners with line number information
+    """
+
+    def __init__(self,cli_options:CliOptions):
+        self.cli_options = cli_options
+    
+    def format_diff(self,diff_content):
+      """
+      Format the diff content in a particular format with corrected line numbers.
+
+      :param: diff_content: str String to format
+      :return: str formatted_string
+      """
+      formatted_diff = []
+      diff_lines = diff_content.splitlines()
+      left = right = 0
+      ll = rl = 0
+      for line in diff_lines:
+        match = re.match(r'^@@ -([0-9]+),([0-9]+) [+]([0-9]+),([0-9]+) @@', line)
+        if match:
+          left = int(match.group(1))
+          ll = len(match.group(2))
```

Comment:
Maybe a bit more explanatory Variable name? 😁

Reply:
Sure! Changed it!